Sources/BraintreePayPal/BTPayPalLineItem.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+
+/// Use this option to specify whether a line item is a debit (sale) or credit (refund) to the customer.
+@objc public enum BTPayPalLineItemKind: Int {
+    /// Debit
+    case debit
+
+    /// Credit
+    case credit
+}
+
+/// A PayPal line item to be displayed in the PayPal checkout flow.
+@objcMembers public class BTPayPalLineItem: NSObject {
+
+    /// Number of units of the item purchased. This value must be a whole number and can't be negative or zero.
+    public let quantity: String
+
+    /// Per-unit price of the item. Can include up to 2 decimal places. This value can't be negative or zero.
+    public let unitAmount: String
+
+    /// Item name. Maximum 127 characters.
+    public let name: String
+
+    /// Indicates whether the line item is a debit (sale) or credit (refund) to the customer.
+    public let kind: BTPayPalLineItemKind
+
+    /// Optional: Per-unit tax price of the item. Can include up to 2 decimal places. This value can't be negative or zero.
+    public let unitTaxAmount: String? = ""
```

Comment:
I think it would be a bit more readable to default these to `nil` instead of empty string. If defaulting to an empty string, and then checking against `""` later on (like line 64) there really is no point of these values being optional types.

Also if a merchant went to go fetch let's say `itemDescription`, when they haven't set it - they would get back an empty string, which I think is more confusing than just a `nil` value since the field is marked as optional.

Let me know if that makes sense / what you guys think?

Reply:
Agreed. I'm gonna change the URL to default to nil as well.

Reply:
I think that makes a ton of sense!